core/accessor_unittest.cpp

```diff
@@ -50,15 +50,15 @@ TEST_F(TestAccessor, FunctionalInMultiThreads) {
         i.init(N);
     for (int i = 0; i < N; i++) {
         workers[i] = new std::thread([&V, i, N, Round, MaxTime, &done]() {
-            unsigned int seed = time(NULL);
+            srand(time(NULL));
```

Comment:
@ddmbr , I just reminded that I changed to use `rand_r` because `lint.py` suggests that. `rand_r` can improve thread-safety. How about reverting this?